mcs/class/Mono.Messaging/Test/AssemblyInfo.cs

```diff
@@ -0,0 +1,60 @@
+﻿//
+// AssemblyInfo.cs
```

Comment:
It's to support `internalsVisibleTo` with the strong-named Mono.Messaging assembly (which I'll explain on your other comment).